src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -7,13 +7,12 @@
     <!-- For this project, we want warnings if there are public APIs/types without properly formatted XML comments (particularly CS1591). -->
     <NoWarn>CS8969</NoWarn>
     <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
-    <IsPackable>true</IsPackable>
     <!-- This library has been annotated to be AOT safe -->
     <EnableAOTAnalyzer>true</EnableAOTAnalyzer>
+    <IsPackable>true</IsPackable>
```

Comment:
Why is this moved/changed? Seems unnecessary.

Reply:
It was moved to group packaging properties together.